src/test/getarg_tests.cpp

```diff
@@ -159,6 +159,98 @@ BOOST_AUTO_TEST_CASE(intarg)
     BOOST_CHECK_EQUAL(m_local_args.GetIntArg("-bar", 11), 0);
 }
 
+BOOST_AUTO_TEST_CASE(patharg)
```

Comment:
In commit "util: Add ArgsManager::GetPathArg() function" (d32f6c0dc89f1a609d59a222b09eabb1eb0da542)

Nice tests!